server/src/controllers/product.controller.ts

```diff
@@ -0,0 +1,27 @@
+import { Request, Response } from "express";
+import Product from "../models/Product/Product";
+
+export const getProducts = async (_: any, res: Response) => {
```

Comment:
Can we please use the standard `req: Request` instead of the underscore here? This will have proper typing for the function. Also please add the return type of the function as well.

Reply:
The req was not required for the getProducts function. It threw an error for declaring it but not using it in the given function. Is it okay if I put _: Request in this?